scala/src/main/org/apache/spark/deploy/csharp/CSharpRunner.scala

```diff
@@ -123,22 +121,23 @@ object CSharpRunner {
           closeBackend(csharpBackend)
         } catch {
           case t: Throwable =>
-            println("[CSharpRunner.main]" + t.getMessage + "\n" + t.getStackTrace)
+            logError(s"${t.getMessage} \n ${t.getStackTrace}")
```

Comment:
I saw there're different stack logging , such as here (stack into logger) to above mentioned (stack may outside)